lib/kane/client.ex

```diff
@@ -32,9 +32,10 @@ defmodule Kane.Client do
 
   defp endpoint, do: Application.get_env(:kane, :endpoint, "https://pubsub.googleapis.com/v1")
   defp token_mod, do: Application.get_env(:kane, :token, Goth.Token)
+  defp account, do: Application.get_env(:kane, :account, :default)
 
   defp auth_header do
-    {:ok, token} = token_mod().for_scope(Kane.oauth_scope())
+    {:ok, token} = token_mod().for_scope({account, Kane.oauth_scope()})
```

Comment:
It would be nice to have a test for when multiple accounts are setup and account is configured.

Reply:
Added some tests for this, they're a bit awkward because we can only assert on the token, but hopefully reasonable.